src/main/drivers/max7456.c

```diff
@@ -484,15 +487,15 @@ bool max7456Init(const max7456Config_t *max7456Config, const vcdProfile_t *pVcdP
 
     uint8_t osdm = max7456Send(MAX7456ADD_OSDM|MAX7456ADD_READ, 0xff);
 
+    __spiBusTransactionEnd(busdev);
```

Comment:
Is this a separate bugfix, and should this go into `4.1-maintenance`?

Reply:
No, it's only applicable to 4.2. The code introduced in #9013 would return without ending the transaction if the max7456 check failed. So if something else shared the bus with the max7456 it would be affected.